packages/safety-suite-api/src/api/models/modelTypes.ts

```diff
@@ -3361,6 +3472,9 @@ export interface TrainingClassFields {
   description?: string;
   dueDate?: number | null;
   endTime?: number | null;
+  expirationInterval?: number | null;
+  expirationIntervalUnits?: string | null;
+  isExpiring?: string | null;
```

Comment:
Just verifying that `isExpiring` is supposed to be a string and not a boolean.

Reply:
Yeah it's one of those stupid `Yes` / `No` enums.